.travis.yml

```diff
@@ -13,7 +15,7 @@ install:
   - pip install flake8==3.0.4
   - pip install -r requirements.txt
   - sudo apt-get update
-  - sudo apt-get install -y linkchecker
+  - sudo apt-get install -y linkchecker libssl1.0.0
```

Comment:
useless: `libssl1.0.0 is already the newest version (1.0.1f-1ubuntu2.22).`

Reply:
https://travis-ci.org/canonical-websites/developer.ubuntu.com/builds/294942510#L954